cromshell

```diff
@@ -437,10 +502,7 @@ function populateWorkflowIdAndServerUrl()
     assertHavePreviouslySubmittedAJob
     WORKFLOW_ID=$(tail -n ${row} ${CROMWELL_SUBMISSIONS_FILE} | head -n 1 | awk '{print $3}' )
     WORKFLOW_SERVER_URL=$(tail -n ${row} ${CROMWELL_SUBMISSIONS_FILE} | head -n 1 | awk '{print $2}' )
-  else
-    # If the user specified anything at this point, assume that it 
-    # is a workflow UUID:
-    if [[ -n "$userSpecifiedId" ]]; then
+	elif [[ $(matchWorkflowId ${userSpecifiedId}) -eq 0 ]] ; then
```

Comment:
indentation is wierd here I think

Reply:
Oh boy.  I forgot to de-tab the file.  Good catch.